### DISCUSSION TITLE
Tensorboard indicators for over-fitting?

### DISCUSSION THREAD

**Discussion by Mangio621** [voicepaw/so-vits-svc-fork#365] [Q&A] [answered]
Hey! I'm currently in the process of training my model and I'm using tensorboard to view my scalars. In this context, I see scalars for grad_norm_d, grad_norm_g, and loss scalars like d/total, g/fm, g/kl, g/lf0, g/mel, g/total and a lr scalar?

What do those mean exactly and what is being measured? I know that loss is very important to look at to indicate over-fitting, but I don't know how to measure over-fitting with these scalars mentioned above. Can anyone kindly give an explanation? Thank you for taking your time to read this. :)

**Comment by 34j** [comment, marked as answer]
Hello. The loss was discussed extensively in the VITS repository, so you can take a look at it. Probably the same parameters are used. I don't have an exact understanding of loss, but I don't think overfitting generally happens that easily (like within 100k steps) (if using default params).

I think it is problematic that the loss is not currently calculated in the validation steps.
